### DISCUSSION TITLE
Template Tarte au citron bandeau cookie

### DISCUSSION THREAD

**Discussion by Wafy-source** [AmauriC/tarteaucitron.js#1200] [General]
Bonjour,

J'aimerais savoir si vous avez à disposition des templates pour le bandeau cookies tarte au citron.

J'aimerais également avoir des explications sur comment modifier le styles du bandeau cookie en version pro.

J'aimerais rendre mon bouton deny all cookies sur ce site https://www.portail-cartegrise.fr/ plus discret en ne l'affichant pas au milieu en ligne avec les autres bouton mais crée un lien continuer sans accepter dans ce style 

<img width="779" alt="Capture d’écran 2024-03-12 à 18 21 10" src="https://github.com/AmauriC/tarteaucitron.js/assets/61818025/80b944ed-578c-4a40-aae9-3e6f6c08be51">

**Comment by AmauriC** [comment]
Pas de template dispo. 
Que ce soit en v pro ou open source, la personnalisation passe par des règles CSS.

L'exemple du bouton "continuer sans accepter" en haut n'est pas ok, même si beaucoup le font, c'est une mauvaise pratique, il faut que les 2 choix soient positionnés pareil.

**Reply by Wafy-source** [reply]
Hello , 

Je sais que continuer sans accepter n'est pas ok mais j'aimerais quand même tester.

Saurais-tu comment je peux faire sur la version pro ? Qu'est ce qu'il faut activer ? Faut-il modifier quelque chose dans tarteaucitron.js ou ajouter un nouveau fichier CSS ?

<img width="847" alt="Capture d’écran 2024-03-12 à 19 19 39" src="https://github.com/AmauriC/tarteaucitron.js/assets/61818025/0299437d-e5b0-4d6e-81d6-f315c95b7125">

A quoi sert ce champ dans la version pro ? Est-ce suffisant pour modifier le style du bandeau ?

**Comment by Wafy-source** [comment]
J'ai trouver un site avec un bandeau que j'aime bien tarte au citron et j'aimerais faire le même design sur le bandeau du milieu pas en bottom.
<img width="1321" alt="Capture d’écran 2024-03-12 à 19 21 27" src="https://github.com/AmauriC/tarteaucitron.js/assets/61818025/a69ecb0f-4651-4d46-87a6-1f023a0b9937">

https://www.cartegrise.com/

Merci

**Reply by AmauriC** [reply]
C'est faisable en CSS, il est tout à fait possible de modifier le positionnement des boutons et les couleurs.

Par contre sur ce point je ne pourrai pas aider vu que ce n'est pas une bonne pratique pour l'utilisateur que je n'ai pas envie de se voir propager, ce n'est pas l'esprit de tarteaucitron ;)

**Comment by Wafy-source** [comment]
Même la méthode de ce site ? 

Les 3 boutons sont sur le même niveau dans ce là : 

![image](https://github.com/AmauriC/tarteaucitron.js/assets/61818025/98b15a0c-16dc-4721-9261-876e138a96e1)

https://www.cartegrise.com/

**Reply by Wafy-source** [reply]
Si vous ne souhaitez pas m'accompagner pour cette méthodes pouvez vous au moins me décrire les étapes pour modifier le bandeau en css en version pro ? 

A quoi sert cet encart : 

<img width="845" alt="Capture d’écran 2024-03-12 à 20 09 17" src="https://github.com/AmauriC/tarteaucitron.js/assets/61818025/c8e1425a-d669-49aa-88e2-995da205182c">

**Reply by AmauriC** [reply]
Je suis le premier à aider et je l'ai déjà fait pour vous.

Par contre la même si c'est le même niveau, le bouton accepter est clairement plus visible que le continuer, le but est bien de faire en sorte que l'utilisateur accepte et ne règle pas un problème technique ou d'accessibilité ?

L'espace css de la V pro sert en effet à injecter du css sur la page.

**Reply by Wafy-source** [reply]
Je ne dit pas que vous ne voulez pas aider mais dans le cas où cette pratique n'est pas la bonne je ne vais pas la faire mais j'aimerais du coupo faire des tests de couleurs par exemple sur mes boutons ou de background du bandeau . 

Avez vous encore un système comme celui ci : 

https://codepen.io/AmauriC/pen/QwaxVP

Dans ce cas là je vois qu'il fallait injecter du css et du html

**Comment by Wafy-source** [comment]
Est-il plus simple de modifier ces 2 fichiers directement ou d'injecter du css dans l'espace css de la v pro ?
<img width="1102" alt="Capture d’écran 2024-03-12 à 20 18 47" src="https://github.com/AmauriC/tarteaucitron.js/assets/61818025/c0a03172-4ab6-4f31-9088-e530e0bf6871">
